src/ProjectTemplates/Web.ItemTemplates/content/RazorView/.template.config/template.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [ "Web", "ASP.NET" ],
+  "name": "Razor View",
+  "generatorVersions": "[1.0.0.0-*)",
+  "description": "Am empty razor view",
+  "tags": {
+    "language": "C#",
+    "type": "item"
+  },
+  "groupIdentity": "Microsoft.AspNetCore.Mvc.RazorView",
+  "precedence": "9800",
+  "identity": "Microsoft.AspNetCore.Mvc.RazorView.8.0",
+  "shortName": "view",
+  "sourceName": "Index",
+  "primaryOutputs": [
+    { "path": "Index.cshtml" }
+  ],
+  "defaultName": "Index",
+  "symbols": {
+    "HostIdentifier": {
+      "type": "bind",
+      "binding": "HostIdentifier"
+    }
+  },
+  "postActions": [
+    {
+      "id": "openInEditor",
+      "condition": "(HostIdentifier != \"dotnetcli\" && HostIdentifier != \"dotnetcli-preview\")",
+      "description": "Opens Index.cshtml in the editor",
```

Comment:
See previous comment regarding this string.

Reply:
fixed ty